### ISSUE TITLE
Extra Item in Deuteronomy Chapter 32

### ISSUE THREAD

**Issue by artsandmetaphysics** [TehShrike/world-english-bible#2]
Thank you very much for publishing this repository.  I am hoping to use the World English Bible in a [Bible quiz game](https://hardbiblequiz.com/) I am working on.  Currently I used the KJV.  I wrote a test that compared the number of verses in each chapter in the KJV to the WEB, and I ran into a few mismatches.  The first mismatch occurred in Deuteronomy Chapter 32.

There is an extra empty item in Deuteronomy Chapter 32.

See https://github.com/TehShrike/world-english-bible/blob/master/json/deuteronomy.json#L7289

**Comment by TehShrike** [comment]
That doesn't look broken to me - e.g. I think there were two spaces between the period and "Give ear..." in the original WEB, and if you compose that JSON version, you would also end up with two spaces.

Is there something specifically that you think is incorrectly transcribed from the official WEB version?

**Comment by artsandmetaphysics** [comment]
I may be misunderstanding how the JSON is meant to be formatted, but I was surprised to see chapter 32 verse 30 section 1 appear before 32 verse 1 section 1.  Also, there is a second 32.30.1 further down.

I had assumed that:

1. Each (chapter, verse, section) combination would be unique
2. The (chapter, verse, section) numbers would increase monotonically

Thank you again for putting this together and publishing it.

**Comment by TehShrike** [comment]
You're right, there shouldn't be two sections with identical chapter/verse/section identifiers.  And it is weird that that first one is out of order at the beginning of the chapter.

I'm not sure what's causing that, but it would be nice if it was fixed.

**Comment by JordanHoffman** [comment]
You're the owner of this repository. What do you mean "it would be nice if it was fixed". This was practically a year ago and it's still not fixed even though it takes 2 seconds to do so. Is this project simply dead?
